### PR TITLE
Add certification menu link in help menu

### PR DESCRIPTION
**Pull request by sam-suresh**
This commit to add https://certification.joomla.org to the Joomla backend help menu.

<h1>how to test:<h1>

Apply this patch
Go to backend Help menu
new Joomla! Certification menu is available

Tq, Joomla! Certification Team

### CODE REVIEW

File: administrator/modules/mod_menu/tmpl/default_enabled.php [hunk 1]
```diff
@@ -349,6 +349,7 @@
 	);
 	$menu->addChild(new JMenuNode(JText::_('MOD_MENU_HELP_RESOURCES'), 'http://resources.joomla.org', 'class:help-jrd', false, '_blank'));
 	$menu->addChild(new JMenuNode(JText::_('MOD_MENU_HELP_COMMUNITY'), 'http://community.joomla.org', 'class:help-community', false, '_blank'));
+	$menu->addChild(new JMenuNode(JText::_('MOD_MENU_HELP_CERTIFICATION'), 'http://certification.joomla.org', 'class:help-certification', false, '_blank'));
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_menu/tmpl/default_enabled.php]: https://certification.joomla.org ?

Comment (zero-24) [inline administrator/modules/mod_menu/tmpl/default_enabled.php]: @sam-sk this side defaults to https can you use it here? So we get `http://certification.joomla.org`

Comment (sam-sk) [inline administrator/modules/mod_menu/tmpl/default_enabled.php]: Sorry. You mean change to https? I just see all other sites http.

**Comment by sam-sk** [comment]
changed to https://certification.joomla.org

**Comment by zero-24** [comment]
Thanks :smile: I have just did a quick look and noticed we have a similiar place for hathor :( Can you fix it there to `/templates/hathor/html/mod_menu/default_enabled.php`?

**Comment by Bakual** [comment]
I don't think it makes sense to add something which is still a work in progress.
Also I don't really see why that should be in the help menu. It's not a help ressource at all.

**Comment by sam-sk** [comment]
Hi @Bakual 
Certification project is now launched and we are accepting registration of Test Centers. It will be useful to have it here to reach out the users. 
Similar to other links there like Joomla Shop etc..

**Comment by Bakual** [comment]
Yeah, the Joomla Shop should be removed imho. It's not related to help either.
But I guess it's the one Joomla "advertising" that got accepted in the backend.

The other links are all related to getting help. Certification doesn't fit into that afaik.

I certainly don't want every ressource that is related to Joomla listed there. with as many subdomains as we currently have, that gets out of hand very fast.

Maybe I'm alone with this view. Anyway, as long as it is still to be launched and no certifications are avalable, there is no point to include a link.

**Comment by sam-sk** [comment]
@Bakual 
Noted. Will re-open once it is launched. 
Thanks

**Comment by infograf768** [comment]
Agree with @Bakual
Certification is unrelated to the CMS help.

**Pull request closed without merging**